### DISCUSSION TITLE
Missing Modules 5.1

### DISCUSSION THREAD

**Discussion by MagRinsvid** [pafuhana1213/KawaiiPhysics#75] [Q&A] [answered]
I thought this was a problem with version 5.0 and the fact that it does not fit UE5.1, but again the same error
![image](https://user-images.githubusercontent.com/117464281/207110856-d998e45a-3968-4470-804b-8a9da1795c7c.png)

![image](https://user-images.githubusercontent.com/117464281/207110302-ee7eeba2-2aa1-44e8-a63d-556e959fa6d6.png)

**Comment by MagRinsvid** [comment, marked as answer]
is this an incorrigible mistake ?

**Reply by shukenmg** [reply]
I think you need to drop it into your project and recompile it (make sure you have a c++ project)

You can also make a new blank c++ project, drop the plugin into it, recompile, then copy the plugin to the engine folder.
